test.sh

```diff
@@ -0,0 +1,41 @@
+PLATFORM=$1
+TYPE=$2
+WHERE="cat != ManualTest"
+TEST_DIR="."
+TEST_PATTERN="*Test.dll"
+ASSEMBLIES=""
+
+if [ -d "$TEST_DIR/_tests" ]; then
+  TEST_DIR="$TEST_DIR/_tests"
+fi
+
+NUNIT="$TEST_DIR/NUnit.ConsoleRunner.3.2.0/tools/nunit3-console.exe"
+NUNIT_COMMAND="$NUNIT"
+NUNIT_PARAMS="--teamcity"
+
+if [ "$PLATFORM" = "Windows" ]; then
+  WHERE="$WHERE && cat != LINUX"
+elif [ "$PLATFORM" = "Linux" ]; then
+  WHERE="$WHERE && cat != WINDOWS"
+  NUNIT_COMMAND="mono --debug --runtime=v4.0 $NUNIT"
+else
+  echo "Platform must be provided: Windows or Linux"
+  exit 1
+fi
+
+if [ "$TYPE" = "Unit" ]; then
+  WHERE="$WHERE && cat != IntegrationTest && cat != AutomationTest"
+elif [ "$TYPE" = "Integration" ] ; then
+  WHERE="$WHERE && cat == IntegrationTest"
+elif [ "$TYPE" = "Automation" ] ; then
+  WHERE="$WHERE && cat == AutomationTest"
+else
+  echo "Type must be provided: Unit, Integration or Automation"
```

Comment:
Thats how I originally did it, but liked this better as it kept everything together. I think if we add more test groups later it would serve as a good reminder to update the echo statement as well.